vue.config.js

```diff
@@ -1,6 +1,7 @@
 const path = require("path");
 
 module.exports = {
+  outputDir: path.resolve(__dirname, "dist/client"),
```

Comment:
moves client build to `dist/client` so final dist structure looks like

```bash
- dist
  - client
  - server
  - shared
  - worker
```

Reply:
Cool